drivers/wifi/simplelink/simplelink_sockets.c

```diff
@@ -58,7 +58,7 @@ static int getErrno(_i32 error)
 		error = EBADF;
 		break;
 #endif
-#if ENSOCK !=  SL_ERROR_BSD_ENSOCK
+#if ENOTSOCK !=  SL_ERROR_BSD_ENSOCK
```

Comment:
Hmm, how this is fixing the compile issue, few lines below we try to set the `ENSOCK` to error variable?
The function seems to try to map TI HAL errno to Zephyr errno, there is also some mention of zephyr in `simplelink/source/ti/drivers/net/wifi/porting/user.h` where it introduces `ENSOCK`. So in theory the `ENSOCK` should exists at this point.
I am not sure how the code should look like but this looks just wrong to me.

Reply:
@cfriedt  ENSOCK is mapped to ENFILE in the hal as @jukkar mentioned, I don't think ENOTSOCK is the correct error code to use when the max number of sockets is exceeded, hence should not be used here.